### ISSUE TITLE
Decide how to convert from UTF8 to ASCII for methods without utf8 support

### ISSUE THREAD

**Issue by andreofner** [andreofner/NER_review#8]
**Solution**: we choose JUnicode library (compared to work best) with the modification that we only end with the first character

**Comment by andreofner** [comment]
example: amyloid-β (Aβ) <-> amyloid-b (Ab) <-> amyloid-beta (Abeta)
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1283364/
http://www.ncbi.nlm.nih.gov/pubmed/16279840

**Comment by jmcejuela** [comment]
Fuck, I have to do this? :P -- Can you please list the methods that do not support utf8 ?

**Comment by andreofner** [comment]
The problem with Abner is solved (by using a wrapper instead of the GUI).
All the other methods at least seem to accept UTF-8 without errors.

**Comment by jmcejuela** [comment]
Nice, issue can be closed?

**Comment by andreofner** [comment]
Right now, all methods run with UTF-8 as input and the shift errors in the results are corrected by me.
This has worked with all single files I used for testing so far.
Testing on a full corpus (Craft, BC3) I have found that even GNormPlus (from NCBI, by far the newest and probably also highest scoring method) returns empty denotation lists for some files. The same files get succesfully annotated when transforming to ASCII.

I have researched a bit and found that
- generally all examples in method documentations are in ASCII
- Encoding issues are never mentioned in publications
- if you search for special characters (e.g. greek letters: "α") in NCBIs PubTator, 100% of the results will contain a transformed ASCII version: (e.g "alpha")
- users testing the taggers semm all to convert to ASCII beforehand, also for example in the BioCreative corpora 

I found in the BioCreative III proceedings:
"We extracted texts from contents of <article-title>, <title>, and <p> elements in the PubMed Central
XML files. We replaced non-ASCII characters into ASCII, e.g., α into alpha, by using a conversion table
because existing NLP tools (e.g., POS tagger and NER tagger) have not been trained with non-ASCII
characters." 
[http://www.biocreative.org/media/store/files/2011/BioCreative_III_Proceedings_.pdf]


The list of methods with UTF-8 shift errors:
Abner
GNormPlus
Gimli
PennBioTagger
